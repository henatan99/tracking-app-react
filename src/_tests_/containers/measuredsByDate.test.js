import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import MeasuredsByDate from '../../containers/measuredsByDate';
import createTestStore from '../factories/storeFactory';

describe('Measureds', () => {
  let store;
  beforeEach(() => {
    store = createTestStore();
  });
  test('should match with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <MeasuredsByDate />
          </MemoryRouter>
        </Provider>,
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
