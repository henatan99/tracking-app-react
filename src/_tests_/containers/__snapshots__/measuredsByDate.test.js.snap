// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Measureds should match with snapshot 1`] = `
<div
  className="measureds-by-date-wrapper"
>
  <div
    className="measureds-date-selector"
  >
    <button
      className="dir-btn"
      onClick={[Function]}
      type="button"
    >
      <span
        className="iconify"
        data-icon="bx:bxs-left-arrow"
        data-inline="false"
      />
    </button>
    <button
      className="dir-btn"
      onClick={[Function]}
      type="button"
    >
      <span
        className="iconify"
        data-icon="bx:bxs-right-arrow"
        data-inline="false"
      />
    </button>
  </div>
  <div
    className="measureds-by-date"
  />
</div>
`;
